recipe/meta.yaml

```diff
@@ -19,14 +19,19 @@ build:
 requirements:
```

Comment:
Remove Lines 14-17

Reply:
I also didn't add that, it was already there.